### ISSUE TITLE
No grave spawned in Nether

### ISSUE THREAD

**Issue by ghost** [FireBall1725/Graves#26]
Testing SSP with FTB Unstable 1.8.9 v3.0.28.
I was surrounded by wither skeletons and blazes in a nether fortress and died. When I went back there was no grave. I even dug out the surrounding blocks. I took a second before respawning so the mod should have had plenty of time.

**Comment by Pihrra** [comment]
The same happened to me in the End. I angered a few too many Endermen and died. When I came back no grave and no items on the ground.

**Comment by FusionLord** [comment]
@charlesbevans can you look at the top of the nether above the bedrock, @Pihrra can you check at the bottom of the world.

**Comment by ghost** [comment]
I checked the nether from 0 to 256 and the corresponding spot in the overworld (per Journey Map). No coffin. However, I did kill myself in the nether and the coffin appeared as expected. I also accidentally teleport myself to y level -100 and the coffin appeared at the bottom bedrock layer so it's obeying bounds checks.

**Comment by Pihrra** [comment]
@FusionLord Sorry for being so late to answer. As soon as I died I went into creative mode and dug down to void level but there was no grave.
However I went to check again before answering and redid the check.
I went down to void level found nothing and died but when I came back again I found my new grave at y:2 (so much higher than where I actually died) so instead I went up from where I originally died and found my grave I thought didn't appear floating in the air at y:253 (I died at y:62)
